spec/service_tests.rb

```diff
@@ -17,31 +17,31 @@ module MarkdownVideos
     youtube: [
       {
         url: "https://youtu.be/StTqXEQ2l-Y",
```

Comment:
I haven't look in details at all your tests, but maybe add more tests testing thing that are weird, malicious, or just straight up not supposed to work.